### DISCUSSION TITLE
Updating Gemfile.lock only

### DISCUSSION THREAD

**Discussion by AnishaKabir** [renovatebot/renovate#26487] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### What is your question?

Is it possible to configure renovate to only update the Gemfile.lock file and not the Gemfile. If so what config settings should I use to achieve this. I enabled lockFileMaintenence, "enabledManagers": ["bundler"], and  "includePaths": ["Gemfile.lock"] in my onboarding pr. However, the pr description says "It looks like your repository dependencies are already up-to-date and no Pull Requests will be necessary right away."

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by rarkins** [comment]
What happens if you merge the PR? Do you get a lock file maintenance PR at the next scheduled time?
